rotatingtext/src/main/java/com/sdsmdg/harjot/rotatingtext/RotatingTextSwitcher.java

```diff
@@ -32,7 +36,7 @@
     private Rotatable rotatable;
 
     private Paint paint;
-
+    Integer[] color_array;
```

Comment:
Is it required to be declared in this class?

Reply:
I have added some functions in the Rotatable class to avoid declaring this array here.